From 68f16ef46890dd08a2d03034ad6f352699282e00 Mon Sep 17 00:00:00 2001
From: Leah Rowe <info@minifree.org>
Date: Thu, 26 Dec 2024 19:45:20 +0000
Subject: [PATCH 1/1] soc/intel/skylake: Don't compress FSP-S

Build systems like lbmk need to reproducibly insert
certain vendor files on release images.

Compression isn't always reproducible, and making it
so costs a lot more time than simply disabling compression.

With this change, the FSP-S module will now be inserted
without compression, which means that there will now be
about 40KB of extra space used in the flash.

Signed-off-by: Leah Rowe <info@minifree.org>
---
 src/soc/intel/skylake/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/soc/intel/skylake/Kconfig b/src/soc/intel/skylake/Kconfig
index c24df2ef75..8e25f796ed 100644
--- a/src/soc/intel/skylake/Kconfig
+++ b/src/soc/intel/skylake/Kconfig
@@ -12,7 +12,7 @@ config SOC_INTEL_COMMON_SKYLAKE_BASE
 	select CPU_SUPPORTS_PM_TIMER_EMULATION
 	select DRIVERS_USB_ACPI
 	select EDK2_CPU_TIMER_LIB if PAYLOAD_EDK2
-	select FSP_COMPRESS_FSP_S_LZ4
+#	select FSP_COMPRESS_FSP_S_LZ4
 	select FSP_M_XIP
 	select GENERIC_GPIO_LIB
 	select HAVE_FSP_GOP
-- 
2.39.5

